### ISSUE TITLE
Resources for getting started with ML

### ISSUE THREAD

**Issue by dan-drews** [dotnet/machinelearning#459]
I am just beginning to try utilizing this framework, and by following the [Github Labeler Sample](https://github.com/dotnet/machinelearning-samples/tree/master/samples/end-to-end-apps/github-labeler) I have been able to build some proof of concept applications within my organization (with surprisingly high accuracy).

However, we are starting to look at taking this a step further. One example is to utilize a StochasticDualCoordinateAscentClassifier along with a PredictedLabelColumnOriginalValueConverter to suggest the top 3 predicted labels, rather than just 1. For example, to not automatically label a github issue, but recommend labels that can be applied.

The description of this project says:

> ML.NET allows .NET developers to develop their own models and infuse custom ML into their applications without prior expertise in developing or tuning machine learning models, all in .NET.

But when I am trying to dig into the documentation, all of the information is very technical, using math and science terms that are way above my understanding. **Are there good resources for people who have high development experience, but no ML experience that I can reference to learn more without feeling like I need a degree in mathematics?** From the little amount of time that I have spent using ML.NET, I am thrilled with the results. Many of our models have had 75% accuracy without much tweaking at all to be necessary, but I am hoping we will be able to use this to take our understanding to the next level.

**Comment by jwood803** [comment]
Hey, @dan-drews! There are a lot of resources out there so I know it can be quite overwhelming. If I may recommend a book to get you on your machine learning journey then I would suggest [Hands-On Machine Learning with Scikit-Learn and TensorFlow](https://www.amazon.com/Hands-Machine-Learning-Scikit-Learn-TensorFlow/dp/1491962291). I really believe it's one of the best books out there.

It is in Python, but the theory and techniques should translate quite well to using ML.NET.

I hope that helps a bit. 😄

**Comment by dan-drews** [comment]
I think the conversation, and the real question I was trying to ask is being discussed in #511 so i will close this out
